RELEASE.md

```diff
@@ -3,6 +3,9 @@
 1. Fixed a bug where rax:roles were not masked correctly if default headers were set.
 1. Added support for ```rax:captureHeader```, this allows setting XPath 3.1 paths at the method request, representation, resource, and resources
    level to allow capturing a header with the result of the XPath. The XPaths can simultaneously introspect headers, uri, methods, and body content (xml or json).
+1. Added support for `X-Relevant-Roles`, a header which will be added to the request any time `rax:roles` are being used.
+   `X-Relevant-Roles` are the values of the `X-Roles` header which match a `rax:roles` value for the resource being requested.
+   In other words, `X-Relevant-Roles` are the user roles which granted access to the resource.
```

Comment:
You should mention that capture head should be enabled for relevant roles to work.